### PR TITLE
Switch to cert-manager managed certificates for AutoticketGenerator webhook

### PR DESCRIPTION
**Pull request by idzikovskyi**

### CODE REVIEW

**Review by skandtandon — APPROVED** [review]
LGTM

File: charts/spark-operator/spark-operator-chart/templates/autotix-certmanager-issuer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{ if .Values.autotix.enable }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ include "autoticket-generator.certManagerIssuer" . }}
```
Comment (mayankgarg-23) [inline charts/spark-operator/spark-operator-chart/templates/autotix-certmanager-issuer.yaml:5]: shouldn't namespace be defined, or we want it in default namespace?

**Comment by mayankgarg-23** [comment]
shoudn't namespace be provided for the Issuer and Certificate resource?

**Comment by mayankgarg-23** [comment]
Is PR created in ezaf repo?

**Comment by idzikovskyi** [comment]
@mayankgarg-23 now, I'm waiting for it to be reviewed here and than I'll open PR to ezaf project.

**Review by mayankgarg-23 — APPROVED** [review]

**Review by sercanCyberVision — APPROVED** [review]

**Pull request merged**